Makefile

```diff
@@ -128,7 +128,7 @@ docker-refresh-dev:
 
 docker-refresh-skus:
 	$(eval VAULT_TOKEN = $(shell docker logs grant-vault 2>&1 | grep "Root Token" | tail -1 | cut -d ' ' -f 3 ))
-	VAULT_TOKEN=$(VAULT_TOKEN) docker compose -f docker-compose.yml -f docker-compose.skus-refresh.yml up -d skus-refresh
+	VAULT_TOKEN=$(VAULT_TOKEN) docker compose -f docker-compose.yml -f services/skus/docker-compose.payment-refresh.yml up -d payment-refresh
```

Comment:
If it's preferred I could rename the file to `docker-compose.skus-refresh.yml`, but I'm not sure if it was used outside of this repo